tests/expectations/tests/replace_template_alias.rs

```diff
@@ -7,7 +7,8 @@
 #[repr(C)]
 #[derive(Debug, Copy, Clone)]
 pub struct Rooted<T> {
-    pub ptr: MaybeWrapped<T>,
+    pub ptr: ::std::os::raw::c_int,
```

Comment:
Not quite a regression since it's really an `int`. Unfortunately there's no way to grab the specialized members from clang. I might take some time to make numTemplateArgs work for this, but until that this is the best we can do.

This was working just by chance, because `MaybeWrapped`'s template argument just happened to be `T`, and we grabbed the top-level type. If instead of that it was `Rooted<U>` -> `MaybeWrapped<U>`, it'd failed (there's a test-case for that).

I guess I can special-case if the spellings are equal to keep the original type, but that's... not really ideal.